python/composio/cli/login.py

```diff
@@ -64,11 +67,23 @@ def _login(
         context.console.print(f"> [green]{url}[/green]")
         if not no_browser:
             webbrowser.open(url)
+
+        # Retrive API key
         code = click.prompt("> Enter authentication code")
-        api_key = Composio.validate_auth_session(
-            key=key,
-            code=code,
+        api_key = Composio.validate_auth_session(key=key, code=code)
```

Comment:
The issue was about validating api key, when the user initialize the SDK - i.e 
```
client = new Composio(api_key="<invalid_api_key"); # should throw error
```

Reply:
Okay, I assumed it was about validating the key on login